testproject-tools-integration/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2021.2.0b4
-m_EditorVersionWithRevision: 2021.2.0b4 (af9ec38f7da3)
+m_EditorVersion: 2021.2.0b12
```

Comment:
Is running on the beta intentional for this project?

Reply:
2021.2 is still in beta I think? I believe the latest is 2021.2.0b13 though.

Reply:
This has been picked up from the version I was using, so which version should this be referencing? I think the test project is on 2020.3? That said I'm currently having issues getting the profiler to run on 2020.3!